### DISCUSSION TITLE
Detect the Source Language

### DISCUSSION THREAD

**Discussion by Aravindan22** [argosopentech/argos-translate#301] [Ideas]
I would like to have a feature to detect the language and I don't know what language is given as source , I have used [langdetect](https://pypi.org/project/langdetect/) library, but the language detection is not as expected, it some times mark English contents as non-English (HTML only) , I would really appreciate to have this feature included

**Comment by omarmhaimdat** [comment]
Hello, you can maybe try this one [whatlang-pyo3](https://github.com/omarmhaimdat/whatlang-pyo3), it's very fast and pretty accurate.
